src/Tests/AzureClientEntryPointTests.cs

```diff
@@ -81,6 +82,94 @@ public async Task FromSnippet()
             Assert.IsNotNull(job);
         }
 
+        [TestMethod]
+        public async Task QIRSubmission()
+        {
+            var entryPointGenerator = Init("Workspace", new string[] { SNIPPETS.HelloQ });
+            var entryPoint = entryPointGenerator.Generate("HelloQ", null, generateQir: true);
+
+            Assert.IsNotNull(entryPoint);
+            var job = await entryPoint.SubmitAsync(
+                new MockQIRSubmitter(new List<Argument>()),
+                new AzureSubmissionContext());
+            Assert.IsNotNull(job);
+        }
+
+        [TestMethod]
+        public async Task ValidParameterTypes()
+        {
+            var entryPointGenerator = Init("Workspace", new string[] { SNIPPETS.ValidParameterTypes });
+            var entryPoint = entryPointGenerator.Generate("UseValidParameterTypes", null, generateQir: true);
+
+            Assert.IsNotNull(entryPoint);
+
+            var validArguments = new Dictionary<Argument, string>()
+            {
+                { new Argument("myBool", new ArgumentValue.Bool(true)), "\"true\"" },
```

Comment:
[code style nit] Dictionary collection syntax is preferred when initializing a dictionary in particular.
```suggestion
                [new Argument("myBool", new ArgumentValue.Bool(true))] = "\"true\"",
```

Reply:
done